seed/ts-sdk/variables/resolved-snippet-templates.md

```diff
@@ -7,3 +7,12 @@ await client.service.post("endpointParam");
 ```
 
 
+```typescript
+import { SeedVariablesClient } from "@fern/variables";
+
+const client = new SeedVariablesClient({ environment: "YOUR_BASE_URL" });
```

Comment:
I think this should contain the other variables in the client constructor. Why doesn't it? Did you generate this? If not, fix the readme generator to include it